src/frontend/packages/core/src/features/cloud-foundry/space-quota-definition/space-quota-definition.component.html

```diff
@@ -0,0 +1,93 @@
+<app-page-header [breadcrumbs]="breadcrumbs$ | async" *ngIf="quotaGuid">
+  <h1>{{ (spaceQuotaDefinition$ | async)?.entity?.name }}</h1>
+</app-page-header>
+
+<div class="space-quota-definition-page">
```

Comment:
We should try and merge this with the org quota page, if we keep them separate there will be duplicate styling. Maybe we could create a quota component that takes configuration to render a quota?